edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -240,6 +236,8 @@ public static async Task<int> MainAsync(IConfiguration configuration)
                                 moduleOwner,
                                 runAsNonRoot));
 
+                        IEnumerable<X509Certificate2> k8sTrustBundle = await CertificateHelper.GetTrustBundleFromEdgelet(new Uri(workloadUri), apiVersion, Constants.WorkloadApiVersion, moduleId, moduleGenerationId);
```

Comment:
what trust bundle is this? the trusted_ca?

Reply:
Yes, same as in IotedgedMode case above - agent needs the trust bundle from above to make X.509 certificate based connections to IoT Hub.

Reply:
but why name k8strustbundle? it is confusing. we can stick to trustbundle itself right.

Reply:
Painful truth?  Adding "k8s" to it made it so I didn't need to move the variable declaration out of the case. Now both Edgelet and K8s mode uses "trustBundle".